src/main/java/com/sourcegraph/cody/agent/WebviewPostMessageParams.kt

```diff
@@ -0,0 +1,47 @@
+package com.sourcegraph.cody.agent
+
+import com.sourcegraph.cody.agent.protocol.ChatError
+import com.sourcegraph.cody.agent.protocol.ChatMessage
+import com.sourcegraph.cody.agent.protocol.ContextFile
+import com.sourcegraph.cody.agent.protocol.PanelNotFoundError
+
+/**
+ * A message sent from the webview to the extension host. See vscode/src/chat/protocol.ts for the
+ * protocol.
+ */
+data class WebviewMessage(
+    val command: String,
+    val text: String,
+    val submitType: String, // One of: "user", "suggestion", "example"
```

Comment:
Is there anything which prevents us from making this proper enum?

Reply:
for now, it's always `"user"`